nodejs-ipaddr.js/nodejs-ipaddr.js.spec

```diff
@@ -0,0 +1,56 @@
+# FIXME
+# Remove nodejs_symlink_deps if installing bundled module with NPM
+
+%global npm_name ipaddr.js
+%global enable_tests 0
+
+%{?nodejs_find_provides_and_requires}
+
+Name: nodejs-%{npm_name}
+Version: 1.2.0
+Release: 1%{?dist}
+Summary: A library for manipulating IPv4 and IPv6 addresses in JavaScript
+License: MIT
+URL: FIXME
+Source0: http://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+BuildRequires: nodejs-packaging
+BuildArch:  noarch
+
+%if 0%{?fedora} >= 19
+ExclusiveArch: %{nodejs_arches} noarch
+%else
+ExclusiveArch: %{ix86} x86_64 %{arm} noarch
+%endif
+
+%{?nodejs_find_provides_and_requires}
+
+%description
+%{summary}
+
+%prep
+%setup -q -n package
+
+%build
+%nodejs_symlink_deps --build
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr .npmignore .travis.yml Cakefile LICENSE README.md bower.json ipaddr.min.js lib package.json src test %{buildroot}%{nodejs_sitelib}/%{npm_name}
+# If any binaries are included, symlink them to bindir here
+
+
+%nodejs_symlink_deps
+
+%if 0%{?enable_tests}
+%check
+%{nodejs_symlink_deps} --check
+#$CHECK
+%endif
+
+%files
+%{nodejs_sitelib}/%{npm_name}
+
```

Comment:
Make it so that it looks like:

```
%files
%{nodejs_sitelib}/%{npm_name}
%doc LICENSE

%files doc
%doc README.md
```

Normally we put the README or doc/ dirs in the doc subpackage, and the LICENSE in the main one. Again npm2rpm could do this with a better default template